samples/parallel-processing-california-housing-data/local_development/.env.example

```diff
@@ -0,0 +1,38 @@
+
+# Azure Subscription
+SUBSCRIPTION_ID = ""
+RESOURCE_GROUP = ""
+
+# Azure ML Workspace/Envirnment/Compute Cluster
+WORKSPACE_NAME = ""
+AML_ENV_NAME=""
+AML_COMPUTE_CLUSTER_NAME = "cpu-cluster"
+AML_COMPUTE_VM_SIZE = "STANDARD_D2_V2"
+AML_COMPUTE_IDLE_TIME = 2400
+AML_COMPUTE_MIN_NODES = 0
+AML_COMPUTE_MAX_NODES = 10
+
+# Azure ML Datastore
+AML_BLOB_DATASTORE_NAME = "blobdatastore"
+AML_STORAGE_ACCOUNT_NAME = ""
+AML_BLOB_CONTAINER_NAME = "azureml"
+AML_STORAGE_ACCOUNT_KEY = ""
+
+# Azure ML Pipeline/Experiment
+PIPELINE_ENDPOINT_NAME = "california_housing_pipeline_endpoint"
+PIPELINE_NAME = "california_housing_pipeline"
+
+# Path
+INPUT_DIR="input"
+SOURCES_DIR_TRAIN = "ml_model"
+PREPARATION_STEP_SCRIPT_PATH = "preparation/preparation_aml.py"
+EXTRACTION_STEP_SCRIPT_PATH = "extraction/extraction_aml.py"
+TRAINING_STEP_SCRIPT_PATH = "training/training_aml.py"
+
+# Azure ML Parallel Run Config for Language Model Pipeline
```

Comment:
You explained the parameters for the parallel step verbally very nice, would be great to have a section in the README where you explain the options in a bit more detail 👍

Reply:
updated in the new commit https://github.com/liupeirong/MLOpsManufacturing/pull/69/commits/5a6dbbf4d8bc97a67a37db14f1a99dc50637c5e2.